### ISSUE TITLE
🐛 Pipeline - Weekly UI Test is failing

### ISSUE THREAD

**Issue by amankumarrr** [SSWConsulting/SSW.Website#1504]
@wicksipedia 

**Describe the bug**

We recently introduced a new pipeline scheduled to run every Monday, but it appears to be broken. Upon reviewing the logs, it indicates that it's pointing to the wrong host URL. We need to address this issue to ensure it runs successfully. Also update the Power Automate flow that checks if the email is received in the mailbox, needs to be updated with correct time zone. 

**Expected behavior**
It should point to production URL

**Screenshots/Video**
<img width="581" alt="image" src="https://github.com/SSWConsulting/SSW.Website/assets/71385247/57a8e30f-72c4-4286-89cc-38576e301638">
**Figure: our pipeline is failing due to wrong host url**

```[tasklist]
### Tasks
- [x] Fix the pipeline
- [x] Update Timezone for the Power Automate flow to AEST
- [x] Done video
```

**Comment by amankumarrr** [comment]
@wicksipedia @bradystroud 

I resolved the pipeline issue by including a default value for `host_url` for both workflow_dispatch and schedule. Additionally, I updated the Power Automate flow to use the `AUS Eastern Standard Time` timezone for both the trigger and date comparisons. Previously, it was set to the default `UTC-International` timezone.

I'll close this issue after recording done video.
